src/pkg/index.ts

```diff
@@ -22,7 +22,8 @@ export const defaultConfig: RouteOptionsCreator = () => {
   return {
     apiBaseUrl: apiBaseUrl,
     kratosBrowserUrl: apiBaseUrl,
-    sdk
+    sdk,
+    baseUrlWithoutTrailingSlash: process.env.BASE_URL || undefined
```

Comment:
@aeneasr Shouldn't this be:
```suggestion
    baseUrlWithoutTrailingSlash: process.env.BASE_PATH || undefined
```
?

I'm not quite sure how this works, though, since the `baseUrlWithoutTrailingSlash` field does not seem to be used anywhere.

Reply:
Also, if I'm supposed to strip the prefix on my reverse proxy, what is `BASE_PATH` going to be used for? Does this application do redirects, where it needs to be aware of its public URL?